README.md

```diff
@@ -1,2 +1,17 @@
 # Pupil Labs Website
 
+## Dependencies & Set Up
+
+**Install node.js and npm on windows**
+
+1. Go to https://nodejs.org/en/ and install node.js (npm included)
```

Comment:
Install instructions are different for Windows vs Posix systems. Just say `install node and npm for your system`